packages/local-cli/link/android/patches/makePackagePatch.js

```diff
@@ -12,8 +12,13 @@ const applyParams = require('./applyParams');
 module.exports = function makePackagePatch(packageInstance, params, prefix) {
   const processedInstance = applyParams(packageInstance, params, prefix);
 
+  const regexp = new RegExp(
+    `,\\n\\s+${processedInstance.replace('(', '\\(').replace(')', '\\)')}`,
+  );
+
```

Comment:
Do we need this `regexp` here? I am sure we could just move it to `revokePatch` file and just match on the `new XXXX` in a line?